KappaAnalysis/interface/KappaEventProviderBase.h

```diff
@@ -36,7 +36,8 @@ class KappaEventProviderBase: public EventProviderBase<TTypes> {
 		m_fi.SpeedupTree(128*1024*1024); // in units of bytes
 
 		// auto-delete objects when moving to a new object. Not default root behaviour
-		m_fi.eventdata.SetAutoDelete(true);
```

Comment:
The line was originally copied from Thomas Hauth, but commented out
https://github.com/artus-analysis/Artus/commit/f1e7b31337c1cf8f2676a694da5d825ed57ae47c#diff-cafa54be8e989291dc681fe0e76685d9R24

Then there's the commit "KappaEventProviderBase.h: small change"
https://github.com/artus-analysis/Artus/commit/24144b419fd068a6523e15ccc26dbcfa1a50bca8

I really can't see why this PR interferes with this settings. I just know that when I run Artus with this PR on multiple files, Artus crashes on the first lumi section change within the first file. I tested running on 1mio files and could not spot any significant growth in memory consumption.